app/models/document.js

```diff
@@ -0,0 +1,32 @@
+module.exports = (sequelize, DataTypes) => {
+  const Document = sequelize.define('Document', {
+    title: {
+      allowNull: false,
+      type: DataTypes.STRING
+    },
+    content: {
+      allowNull: false,
+      type: DataTypes.TEXT
+    },
+    OwnerId: DataTypes.INTEGER,
+    access: {
+      defaultValue: 'public',
+      type: DataTypes.STRING,
+      validate: {
+        isIn: [['private', 'public', 'role']]
+      }
+    }
+  }, {
+    classMethods: {
+      associate: (models) => {
```

Comment:
**Nitpick - Code Quality**
You want to make use of object method shorthand as stated in our style guide.

In this case, you'll have:
```
classMethods: {
  associate(models) {
    ...
  }
}
```

Take a look at the airbnb style guide:
https://github.com/airbnb/javascript#objects